xpk.py

```diff
@@ -2519,16 +2519,26 @@ def workload_delete(args) -> int:
   elif not will_delete:
```

Comment:
Can add a quick detail about the cool batching in the readme? TY

Reply:
Done